README.md

```diff
@@ -1,7 +1,83 @@
-# Polymake
+# Polymake.jl
 
-[![Build Status](https://travis-ci.org/sebasguts/Polymake.jl.svg?branch=master)](https://travis-ci.org/sebasguts/Polymake.jl)
+Julia package for using [polymake](https://polymake.org/doku.php), a software for research in polyhedral geometry.
+This package is developed as part of the [OSCAR](https://oscar.computeralgebra.de) project.
 
-[![Coverage Status](https://coveralls.io/repos/sebasguts/Polymake.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/sebasguts/Polymake.jl?branch=master)
+## Installation
 
-[![codecov.io](http://codecov.io/github/sebasguts/Polymake.jl/coverage.svg?branch=master)](http://codecov.io/github/sebasguts/Polymake.jl?branch=master)
+The current version relies on an unreleased version of polymake. A compatible version is available at [polymake/polymake#snapshots](https://github.com/polymake/polymake/tree/Snapshots) and it can be compiled as follows where `GIT_FOLDER` and `INSTALL_FOLDER` have to be substituted with your favorite folder.
```

Comment:
Please add a link to the Polymake website showing all the dependencies one needs to compile polymake

Reply:
good point added it.